OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/AndroidICCAnalysis.scala

```diff
@@ -1387,10 +1404,12 @@ object ManifestParsingKey extends ProjectInformationKey[(Map[String, ListBuffer[
 /**
  * Schedules the execution of an AndroidICCAnalysis.
  * In order for the analysis to work the path to the projects AndroidManifest must be set via setManifest.
```

Comment:
This comment is really missing an imporant comma: `to work, the path`. Though I don't think it is still valid, now that the manifest is parsed by a project information key?